### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by xaleehrtyunus**
## Portfolio: project setup and mobile skeleton
In this project;
- I created a mobile version of my portfolio 
- I added icons
- I added text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
-# portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> One paragraph statement about the project.
+
+![screenshot](./app_screenshot.png)
```
Comment (tahiry-dev) [inline README.md]: - Your screenshot couldn't be shown, would you please fix it?

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
...
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```
Comment (tahiry-dev) [inline README.md]: - It is better to inform your visitor how to run your project locally

File: css/styles.css [hunk 3]
```diff
@@ -1,9 +1,9 @@
 .body-section {
-    padding-top: 20%;
-    width: 50%;
-    margin: 0 auto;
+  padding-top: 20%;
+  width: 50%;
+  margin: 0 auto;
 }
```
Comment (tahiry-dev) [inline css/styles.css]: - For this project, we need to use Flexbox to align element please

File: index.html [hunk 4]
```diff
@@ -38,18 +38,34 @@
...
-			<img class="icons" src="./images/Ellipse 4.png" alt="circle">
-			<img class="icons" src="./images/Ellipse 3.png" alt="circle">
-
+		<div class="d-flex  mt-5">
+			<img class="icons" src="./images/Rectangle.png" alt="circle">
+			<img src="./images/cycle.png" alt="">
+		</div>
+		<div>
+			<img class="icons" src="./images/half.png" alt="circle">
+			<img class="icons" src="./images/smallcycle.png" alt="circle">
```
Comment (tahiry-dev) [inline index.html]: -Those images are there for decoration. I think it is better to use them as a background image.

**Review by tahiry-dev — CHANGES_REQUESTED** [review]
Hello xaleehrtyunus👋

You did a great job but there are few issues you need to update before moving to the next project

I appreciate your effort to implement bootstrap in your project 🙏, but for this section, and for this project here are the general requirements:

- Understand how to parse a Figma design to create a UI.
- Use Flexbox to place elements on the page.
- Use images and backgrounds to enhance the look of the website.

Consequently, using bootstrap might prevent you to fulfill those requirements, so I suggest you using HTML-CSS only

## Required Changes ♻️

_Kindly check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Portfolio</title>
+	
+	<link rel="stylesheet" href="./styles.css">
+    </head>
+  <body>
+		<header class="nav">
+			<a class="logo" href="#">My logo</a>
```
Comment (SajjadAhmad14) [inline index.html:12]: [OPTIONAL] Please follow the exact font and color defined in the design.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+			<img src="./images/menu.png" alt="menu-icon">
+		</header>
+
+			<section class="body-section circles">
+				
+				<h1 class="main-text">I'M ZALIHA GLAD TO SEE YOU</h1>
+				<p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```
Comment (SajjadAhmad14) [inline index.html:19]: [OPTIONAL] Please follow the exact font and color defined in the design.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+			</section>
+			<a href="#" class="connect">LET'S CONNECT</a>
```
Comment (SajjadAhmad14) [inline index.html:21]: [OPTIONAL] Please follow the exact font and color defined in the design.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Portfolio</title>
+	
+	<link rel="stylesheet" href="./styles.css">
+    </head>
+  <body>
+		<header class="nav">
```
Comment (SajjadAhmad14) [inline index.html:11]: [OPTIONAL] Header on the web page has a large margin from the top. Please remove it.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

## Hi Zaliha Gambo Yunusa, 
### Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
### OPTIONAL BUT RECOMMENDED
- [ ] Please see inline comments.
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**